lib/_debug_agent.js

```diff
@@ -45,18 +47,21 @@ exports.start = function start() {
 
 function Agent() {
   net.Server.call(this, this.onConnection);
+  this.maxConnections = 1;
```

Comment:
Perhaps add a comment about that, to someone unfamiliar with the entire `net` Server API this will be confusing.